src/view/utils.styl

```diff
@@ -26,6 +27,10 @@ $gap = 8px;
   margin-top: $gap*2 !important;
 }
 
+.u-gapTopHalf {
+  margin-top: $halfGap !important;
+}
```

Comment:
Did you try aligning the element using `u-alignItemsCenter` and/or `u-justifyContentCenter` classes instead? If you can't get the alignment right without this class, let me know. I'd like to take a peek and see if we can get rid of `u-gapTopHalf`.

Reply:
To be honest, my CSS knowledge harks back to when I had to support IE6 so I never learned how flex works. I tried u-justifyContentCenter, but it centers across the whole paragraph and not just the first line. See the asterisk centering in this screenshot:
<img width="574" alt="Screen Shot 2020-09-15 at 3 31 31 PM" src="https://user-images.githubusercontent.com/952132/93267463-f490a100-f768-11ea-9532-7081e2b71c6c.png">

You are welcome to try to get rid of gapTopHalf, or I can just add margin in the local stylesheet for the noSelectedNodeView component if you don't want it as a utility.

Reply:
Thanks for the explanation and screenshot. I misunderstood what you were trying to accomplish. With that in mind...meh, I'm good with it as it is or in a local stylesheet. 👍